Order group editor object permissions (#10469)
wagtail/wagtail: docs/extending/customising_group_views.md

    ...,
]
```

> ## Customising the Group Editor Object Permissions Order

Reviewer: ```suggestion
## Customising the group editor permissions ordering
```

Just simplifying and removing the article casing on the titles, we usually avoid this and just title case proper nouns.

Author: Changed in [1c515e4](https://github.com/wagtail/wagtail/pull/10469/commits/1c515e4d49262d2a8ed80eb201fa8046670563be)

> The order that object types appear in the Group Editor "Object permissions" and "Other permissions" sections can be configured by registering that order in one or more AppConfig definitions. The order value is typically an integer between 0 and 999, although this is not enforced.

Reviewer: ```suggestion
The order that object types appear in the group's "Object permissions" and "Other permissions" sections can be configured by registering that order in one or more `AppConfig` definitions. The order value is typically an integer between 0 and 999, although this is not enforced.
```

Just avoiding the 'Group Editor' term, we do not really use that in this page.

Author: Changed in [1c515e4](https://github.com/wagtail/wagtail/pull/10469/commits/1c515e4d49262d2a8ed80eb201fa8046670563be)

```python
from django.apps import AppConfig

class MyProjectAdminAppConfig(AppConfig):
    name = "myproject_admin"
    verbose_name = "My Project Admin"

    def ready(self):
        from wagtail.users.permission_order import register

        register("gadgets.SprocketType", 150)
        register("gadgets.ChainType", 151)

>         register("site_settings.Settings", 160)

Reviewer: Should we use a kwarg here? I know this is not a hook but all other hooks with ordering take their order as a keyword arg. Just lets us add more options later if needed.

```suggestion
        register("site_settings.Settings", order=160)
```

Author: Changed in [eb0f1a5](https://github.com/wagtail/wagtail/pull/10469/commits/eb0f1a5aeccb835e7c0d75661cfbbb621d178447)
